lib/core/extension/proposal_model_extensions.dart

```diff
@@ -30,5 +32,6 @@ extension ProposalModelTimeFormatting on ProposalModel {
   double unityToPercent() => unity==null?0:unity!*.01;
   double commitmentToPercent() => commitment == null ? 0 : commitment! * .01;
   bool isExpired() => expiration!.toLocal().isBefore(DateTime.now());
+  Color percentageColor()=> quorumToPercent()>=.2 && unityToPercent()>=.8 ?HyphaColors.success:HyphaColors.error;
```

Comment:
it would be nice to have an "isPassing" function for this on proposal model

IsPassing should be interesting not just for color, but also for other purposes. 

Also, the quorum percentage numbers are settings on the DAO, they can't be hard-coded. 

They need to be loaded from the DAO

Since this is a UX change, I will approve and leave it up to you if you want to fix one or both of these items in this PR or make a todo and fix it in another PR.

Reply:
Thank you for the feedback @n13 ! Regarding the quorum percentage numbers, in our code, we don't hard-code them. We actually receive the percentages for both quorum and unity from the GraphQL API. The only adjustment we make is reformatting them from int to double, as the package we use for displaying percentages requires double as input, not int.

As for the isPassing function, I agree that it would be a good idea to implement it for broader use cases beyond just color. I'll handle it in this PR

Reply:
There's 2 values in here, 0.2 and 0.8, which are hard-coded. 

Not sure if you're referring to other places in the code where these values are loaded from the server, but they need to be loaded from the server also here, when we indicate fail/pass state ;)